Makefile

```diff
@@ -11,10 +11,10 @@ main.o: main.cpp
 	$(CC) $(CFLAGS) -c main.cpp
 
 character.o: character.cpp
-	$(CC) $(CFLAGS) -c character.cpp
+	$(CC) $(CFLAGS) -c character.cpp character.h
```

Comment:
A character nem hasznalja a parser-t? ha igen, annak is kellene itt a headerje, nem?